core/README.md

```diff
@@ -30,8 +30,8 @@ The core of a STAC API is its landing page, which is the starting point to disco
         }
     ],
     "conformsTo" : [
-        "http://stacspec.org/spec/api/1.0.0-beta.1/core",
-        "http://stacspec.org/spec/api/1.0.0-beta.1/stac-search"
+        "http://api.stacspec.org/1.0.0-beta.1/core",
```

Comment:
It seems there are some places that are missing the `v` before the version number.